### PR TITLE
Document update lock & bind mount restrictions

### PR DESCRIPTION
**Pull request by cywang117**

### CODE REVIEW

**Comment by cywang117** [comment]
@vipulgupta2048 I requested a review from you just in case. Let me know if I should self-serve these changes in the future 👍🏼

File: pages/learn/develop/multicontainer.md [hunk 1]
```diff
@@ -89,6 +89,8 @@ services:
 
 For devices upgraded from older versions of {{ $names.os.lower }} to v2.12.0 or higher, a link will automatically be created from the `/data` directory of the container to the `resin-data` named volume (similar to above). This ensures fleet behavior will remain consistent across host OS versions. One notable difference is that accessing this data via the host OS is done at `/var/lib/docker/volumes/<FLEET ID>_resin-data/_data`, rather than the `/mnt/data/resin-data/<FLEET ID>` location used with earlier host OS versions.
 
+{{ $names.company.lower }} does not support the use of bind mounts at this time, aside from the ones which are provided by feature labels.
```
[the remark this answers is not in the document]
Comment (cywang117) [reply pages/learn/develop/multicontainer.md:92]: This is referring to `io.balena.features.supervisor-api`, etc. There is a section in this same document right below this addition titled `Labels` that goes over the labels more in depth. I'll add a link to that section.

[the remark this answers is not in the document]
Comment (cywang117) [reply pages/learn/develop/multicontainer.md]: Added the link!

**Comment by cywang117** [comment]
@balena-ci I self-certify!

**Comment by vipulgupta2048** [comment]
> I requested a review from you just in case. Let me know if I should self-serve these changes in the future 👍🏼

You can absolutely @cywang117 If you are the product owner or there is a degree of confidence in the changes you are making or even if the change is small then you can go ahead and auto-merge without asking for review or second looks. 

For anything else, sure happy to review but still unblocked from merging if needed.

**Pull request merged**